draft/implementation-notes/index.html

```diff
@@ -489,6 +489,11 @@ <h2>Version Numbering</h2>
           following <code>v</code> is a zero then the number format is zero padded to fixed length, otherwise it
           is simply an integer.
         </p>
+        <p>
+          Systems with version directories have often used zero padding in order to show version order with common
+          lexical sorting tools (such as Unix <code>ls</code>). Zero padding is not recommended in order to avoid
+          having to make arbitrary choices of padding length or to place limits of the number of versions supported.
```

Comment:
typo?
Should the `of` be `on`:
"place limits [of] the number of versions"

Reply:
Indeed, will fix